server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -140,6 +140,7 @@ export class BasicRestWrapper extends RestWrapper {
                         debug(`request to ${options.url} failed ${error ? error.message : ""}`);
                     }
 
+                    // TODO: replace retryAfter with retryAfterMs
```

Comment:
Nit: Can we open an issue instead of sprinkling `// TODO` comments in the code?  Too often they never get cleaned up and it's distracting and erodes confidence, IMO.

Reply:
Opened #9907 and assigned it to myself.